camera_calibration/nodes/cameracalibrator.py

```diff
@@ -95,7 +95,7 @@ def run(self):
 
 class CalibrationNode:
     def __init__(self, boards, service_check = True, synchronizer = message_filters.TimeSynchronizer, flags = 0,
-                 pattern=Patterns.Chessboard, camera_name='', checkerboard_flags = 0):
+                 pattern=Patterns.Chessboard, camera_name='', folder_location='/tmp/', checkerboard_flags = 0):
```

Comment:
please use the tempfile module (and gettempdir()) so that it can work on multiple OSes (and yes, the current implementation is definitely not working on other OSes :) ).